styles/styles.go

```diff
@@ -37,5 +46,42 @@ func JoinVertical(top, bottom string, height int) string {
 	return top + fill + bottom
 }
 
-//go:embed theme.json
-var Theme []byte
+// CustomTheme reads in a custom glamour theme configuration
+// from a filepath
+func CustomTheme(filepath string) glamour.TermRendererOption {
+	if fileExists(filepath) {
+		return glamour.WithStylesFromJSONFile(filepath)
+	}
+	switch filepath {
+	case "default":
+		return glamour.WithStylesFromJSONBytes(DefaultTheme)
+	default:
+		return glamour.WithStylesFromJSONBytes(DefaultTheme)
+	}
+}
+
+// SelectTheme picks a glamour style config based
+// on the theme provided in the markdown header
+func SelectTheme(theme string) ansi.StyleConfig {
+	switch theme {
+	case "ascii":
+		return glamour.ASCIIStyleConfig
+	case "light":
+		return glamour.LightStyleConfig
+	case "notty":
+		return glamour.NoTTYStyleConfig
+	default:
+		return glamour.DarkStyleConfig
+	}
+}
+
+// fileExists is a helper to verify
+// that the provided filepath exists
+// on the system
+func fileExists(filepath string) bool {
+	info, err := os.Stat(filepath)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return !info.IsDir()
+}
```

Comment:
Totally optional but we can create a package called `file` and make this a method `Exists`: `file.Exists(path)`